dataset.py

```diff
@@ -254,6 +259,33 @@ def partitioned_reference_data(reference_csv_filename: str) -> PartitionedDatase
   print_split(partition_data)
   return partition_data
 
+def load_reference_samples(filters: list[ee.Filter] = []) -> pd.DataFrame:
+  '''
+  Given an optional list of filters, returns a DataFrame containing all current
+  reference sample from Earth Engine. You must have the proper authorization
+  to access this data, which is obtained by belonging to an organization added
+  to TimberID.org
+  '''
+  import google
+  from google.colab import auth
+  auth.authenticate_user()
+
+  credentials, project_id = google.auth.default()
+  ee.Initialize(credentials, project='river-sky-386919')
+  fc = ee.FeatureCollection('projects/river-sky-386919/assets/timberID/trusted_samples')
+  for filter_fc in filters:
+    fc = fc.filter(filter_fc)
+  info = fc.getInfo()
+  features = info['features']
+  dictarr = []
+
+  for f in features:
+      attr = f['properties']
+      attr[_LATITUDE_COLUMN_NAME] = f['geometry']['coordinates'][1]
+      attr[_LONGITUDE_COLUMN_NAME] = f['geometry']['coordinates'][0]
```

Comment:
Maybe a dumb questions, but are we sure it's lon/lat not lat/lon?

Reply:
yes, I double checked.  I will triple check, by comparing the reference data with what we expect.

Reply:
triple checked and I added some minimal output that shows 5 example lat/longs as well as another assert that ensure min(long) < -70.